src/arm/midr.c

```diff
@@ -208,6 +266,54 @@ struct features* get_features_info(void) {
   feat->NEON = true;
   feat->SVE = false;
   feat->SVE2 = false;
+#elif defined _WIN32
+
+  // CP 4020 maps to the ID_AA64PFR0_EL1 register on Windows
+  // https://developer.arm.com/documentation/ddi0601/2024-06/AArch64-Registers/ID-AA64PFR0-EL1--AArch64-Processor-Feature-Register-0
+  int64_t pfr0 = 0;
+  if(!get_win32_core_info_int(0, "CP 4020", &pfr0, true)) {
+    printWarn("Unable to retrieve PFR0 via registry");
+  }
+  else {
+
+    // FP and AdvSimd are signed. "-1" is the only value that indicates
+    // the feature is not available
+
+    // FP[19:16]
+    // 0: Float support
+    // 1: Float support + FP16
+    int8_t fp = ((int64_t)(pfr0 << (60 - 16)) >> 60);
+
+    // AdvSimd[23:20]
+    // 0: AdvSimd support
+    // 1: AdvSimd support + FP16
+    int8_t adv_simd = ((int64_t)(pfr0 << (60 - 20)) >> 60);
+    feat->NEON = (adv_simd >= 0) && (fp >= 0);
```

Comment:
Is this `(adv_simd >= 0) && (fp >= 0)` documented somewhere? How do you know this will always return true for NEON and false for non NEON? I'm also surprised there is no direct way to get if NEON is supported, just like AES, SHA1...